Makefile.dep

```diff
@@ -58,6 +58,20 @@ ifneq (,$(filter ng_ipv6_hdr,$(USEMODULE)))
   USEMODULE += ng_pktbuf
 endif
 
+ifneq (,$(filter ng_ipv6_router,$(USEMODULE)))
+  USEMODULE += ng_ipv6
+  USEMODULE += ng_fib
```

Comment:
You're right. Since I kept routing out for now (and it will become part of the neighbor discovery anyways), this whole `ifneq` can be removed. Will do.